src/models/reranking.rs

```diff
@@ -41,6 +53,7 @@ pub struct RerankerModelInfo {
     pub description: String,
     pub model_code: String,
     pub model_file: String,
+    pub additional_files: Vec<String>,
```

Comment:
Hopefully no one is constructing this type themselves. It'll break for them.
I am fine with this.